tfjs-backend-webgpu/src/webgpu_util.ts

```diff
@@ -58,6 +58,26 @@ export function computeDispatch(
   ];
 }
 
+export function computeWorkPerThreadForMatMul(
```

Comment:
Is this function being used anywhere else? Maybe it would make sense to just unroll this logic within `matmul_packed_webgpu` - otherwise it's a little bit confusing to follow the logic, especially because this function accepts `workPerThread` as an argument while calculating a new `workPerThread`.

Reply:
Done. PTAL